engine/src/test/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActorSpec.scala

```diff
@@ -59,7 +63,7 @@ class WorkflowExecutionActorSpec extends CromwellTestkitSpec with BeforeAndAfter
       val isFatal = (_: Throwable) => false
 
       // Sleep a bit, and let the metadata do its thing in the background.
-      Thread.sleep(1000L)
+      Thread.sleep(3.seconds.dilated.toMillis)
```

Comment:
I'm hoping that as I slowly remove increasingly less low hanging fruit (I believe this is high hanging fruit but we'll see) that the reduced load will make some of the higher hanging fruit more stable